contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
```

Comment:
Regarding 1st level triage.  We're doing well.  Consider that there were [300+ issues](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20created%3A%3E2017-07-01%20) opened in July 2017 and only [28 of them](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue%20created%3A%3E2017-07-01%20label%3Aneeds-sig) lack a sig label.  The automation and prompting is working at driving the issues to sigs.  

This does not address the miscategorization problem, nor does it address the 2nd level triage.  But it's worth noting that we are getting people to apply sig labels.